bridgev2/database/upgrades/21-disappearing-message-fkey.postgres.sql

```diff
@@ -0,0 +1,7 @@
+-- v21: Add foreign key constraint from disappearing_message.mx_room to portals.mxid
+CREATE UNIQUE INDEX portal_bridge_mxid_idx ON portal (bridge_id, mxid);
+ALTER TABLE disappearing_message
```

Comment:
Postgres also needs to delete the extra rows if there are any

```suggestion
DELETE FROM disappearing_message WHERE mx_room NOT IN (SELECT mxid FROM portal WHERE mxid IS NOT NULL);
ALTER TABLE disappearing_message
```